new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Operator.java

```diff
@@ -40,4 +52,53 @@ public static boolean uninstall(HelmParams params) {
   public static boolean scaleDomain(String domainUid, String clusterName, int numOfServers) {
     return true;
   }
+
+  /**
+   * Image Name for the Operator. Uses branch name for tag in local runs
+   * and branch name, build id for tag in Jenkins runs.
+   * @return image name
+   */
+  public static String getImageName() {
+    String image = "";
+    String imageName = System.getenv("IMAGE_NAME_OPERATOR") != null
```

Comment:
Can you Optional.ofNullable(...).orElse(...) pattern here too.

Reply:
done